backend/api/Program.cs

```diff
@@ -45,10 +47,15 @@
     }
 );
 
-builder.Configuration.AddAzureKeyVault(
-    new Uri(builder.Configuration.GetSection("KeyVault")["VaultUri"]),
-    new DefaultAzureCredential()
-);
+// Adds keyvault as config option if environment is set to production
+// This can be changed locally in the "Properties/launchSettings.json" file
+if (builder.Environment.IsProduction())
+{
+    builder.Configuration.AddAzureKeyVault(
+        new Uri(builder.Configuration.GetSection("KeyVault")["VaultUri"]),
+        new DefaultAzureCredential()
+    );
+}
```

Comment:
I'm not certain this is the preferred solution. Mainly because we're now using a different specific system for secrets (secret manager) locally when it doesn't really solve our problem which is to run the application without secrets in an open source fashion. It might be this is not really that easy with this application as it was with ISAR. 

I suggest we go back to having the keyvault as default also in development. Another error that occurs with this change is that there are no keyvaults if you use the suggestes launch.json change to production as there is no appsettings for production yet.

Reply:
Easier to use keyvault also for local development then and only require the user to manage one secret.